gear/gear/database.py

```diff
@@ -235,10 +236,15 @@ async def execute_update(self, sql, args=None):
         async with self.conn.cursor() as cursor:
             return await cursor.execute(sql, args)
 
-    async def execute_many(self, sql, args_array):
+    async def execute_many(self, sql, args_array, query_name=None):
```

Comment:
Just to be clear, we're only monitoring `execute_many`. Is there a plan to add more monitoring of queries in the future?

Reply:
Right now we do the following:
1. Count the total number of sql transactions performed per second
2. Time explicitly monitored queries

In terms of timing more queries, it's currently on an as-needed basis. It felt potentially overbearing to require every query have a unique name and add timing, rather we've been targeting hot-path queries as needed to address big performance issues.

It could be nice to have a more complete sql monitoring solution in place, but I think would require some more thought to make it easy to analyze from an operational perspective.